src/main/java/com/ericsson/ei/handlers/RMQHandler.java

```diff
@@ -185,4 +216,63 @@ private boolean isRMQCredentialsSet() {
         String bindingKeysWithoutWhitespace = bindingKeys.replaceAll("\\s+", "");
         return bindingKeysWithoutWhitespace.split(",");
     }
+
+    /**
+     * This method is used to delete the bindings in rabbitMQ.
+     * By comparing the binding keys used in the properties and binding keys stored in mongoDB.
+     * newBindingKeysArray is the only binding keys array.
+     * AMQPBindingObjectList is entire list of bindings.
+     * Binding key which is not present in the current AMQPBindingObjectList gets deleted and removed from mongoDB.
+     * @return
+     */
+
+    private void deleteBindings(String[] newBindingKeysArray, List<Binding> AMQPBindingObjectList) {
+        // Creating BindingKeys Collection in mongoDB
+        ArrayList<String> allDocuments = mongoDBHandler.getAllDocuments(dataBaseName, collectionName);
+        ArrayList<String> existingBindingsData = new ArrayList<String>();
+        if (!allDocuments.isEmpty()) {
+            for (String bindings : allDocuments) {
```

Comment:
I wonder if this method should perhaps be split into several smaller methods, if possible. It doesn't affect functionality but it could improve readability of the code. It's up to you if you want to refactor it or not.